pkg/upgrade/control_plane.go

```diff
@@ -859,6 +859,20 @@ func generateReplacementMachineName(base, upgradeID string) string {
 	return machineName + machineSuffix
 }
 
+func getAllButMachineOwners(ownerRefs []metav1.OwnerReference) []metav1.OwnerReference {
+	filtered := make([]metav1.OwnerReference, 0, len(ownerRefs))
+	for _, owner := range ownerRefs {
+		if owner.Kind == "Machine" {
+			// Skip Machine owners so that it will be adopted by the
+			// MachineController
```

Comment:
Nit: Is the purpose to filter out owners of kind Machine? If it is, this comment is unnecessary as it confuses the reader (me, at least) and makes them think if there's any other purpose of this very simple function.